packages/bento-design-system/src/Charts/DonutChart/DonutChart.tsx

```diff
@@ -0,0 +1,83 @@
+import {
+  PieChart as RechartPieChart,
+  Pie,
+  Cell,
+  Tooltip,
+  Legend,
+  ResponsiveContainer,
+} from "recharts";
+import { useBentoConfig } from "../../BentoConfigContext";
+import { bodyRecipe } from "../../Typography/Body/Body.css";
+import { allColors } from "../../util/atoms";
+import { ChartProps } from "../ChartProps";
+import { legendContent } from "../Legend/Legend";
+import { tooltipContent } from "../Tooltip/Tooltip";
+
+type Props<D extends string, C extends string> = ChartProps & {
+  data: Record<D, unknown>[];
```

Comment:
shouldn't this include also `C` as keys?
```suggestion
  data: Record<C | D, unknown>[];
```

Reply:
yes, it should